qa/suites/fs/snaps/conf

```diff
@@ -0,0 +1 @@
+.qa/cephfs/conf
```

Comment:
This suite does not exist in luminous.

Reply:
@batrick Not sure what you mean here. There does seem to be an `fs/snaps` suite in luminous: https://github.com/ceph/ceph/tree/luminous/qa/suites/fs/snaps

Reply:
Sorry I thought it was `fs/upgrade`, nevermind.